mirrormanager2/templates/fedora/statistics.html

```diff
@@ -0,0 +1,25 @@
+{% extends "master_noauth.html" %}
+
+{% block title %}Statistics{% endblock %}
+{%block tag %}statistics{% endblock %}
+
+{% block content %}
+<h1>Statistics</h1>
+<p class="centered">Mirrorlist access breakdown by
+<a href="{{url_for('statistics')}}/{{ today }}/countries">country</a>
+| <a href="{{url_for('statistics')}}/{{ today }}/archs">architecture</a>
+| <a href="{{url_for('statistics')}}/{{ today }}/repositories">repository</a>
+{{today}}</p>
```

Comment:
I'd be in favor of adding a little text before the date, otherwise it kind of looks like ``repository <date>``, maybe ``on <date>``?